### ISSUE TITLE
Delphi paper question.

### ISSUE THREAD

**Issue by qinyiqun** [mc2-project/delphi#20]
The baseline accuracy of Resnet32 in your paper is only 68% percent, but it should be much higher , maybe 75% now. And in your code, The Resnet32 structure is [5, 5, 5] with lower dimensions. Is it the reason or something else.
If it is because the structure, i tried to use pytorch to repeat the 68% accuracy on your 5, 5, 5 Resnet32, but only get 65.7%.

**Comment by ryanleh** [comment]
We used the Tensorflow Keras Resnet32 implementation at the time of paper writing and haven't updated things since then. I'm not sure why PyTorch wouldn't achieve the same accuracy we got, maybe try using Tensorflow or playing with your hyper-parameters?

**Comment by qinyiqun** [comment]
Ok, i will try it.
